cmd/get/get_schedulers.go

```diff
@@ -79,7 +79,7 @@ func (cs *GetSchedulers) run(_ *cobra.Command, args []string) error {
 	return nil
 }
 
-func (cs *GetSchedulers) printSchedulersTable(schedulers []*v1.Scheduler) {
+func (cs *GetSchedulers) printSchedulersTable(schedulers []*v1.SchedulerWithoutSpec) {
```

Comment:
I really dislike this name SchedulerWithoutSpec but I understand that it's an import from "maestro" :)